source/js/admissions.js

```diff
@@ -22,8 +15,7 @@ $(modalCloseBtn).click((e) => {
   Handler.hide();
 });
 
-// var btn = Viewer._addButton("modal-inner-close", "Back");
-// - Fix event listener getting ducked up when shown
+// var btn = Viewer._addButton("modal-inner-close", "Back"); - Fix event listener getting ducked up when shown
```

Comment:
why not just remove?

Reply:
I'll leave it there for now, it's a reminder for me to have a look at it and fix it. I may remove it later tho.